cpu/cc2538/periph/i2c.c

```diff
@@ -26,33 +27,33 @@
 
 #include "mutex.h"
 #include "cpu.h"
+#include "periph/gpio.h"
 #include "periph/i2c.h"
-#include "thread.h"
 #ifdef MODULE_XTIMER
 #include "xtimer.h"
-#endif
+#else
+#include "thread.h"
```

Comment:
I won't NACK this PR for this change but I don't see what this has to do with the i2c implementation